OneCommunityGlobal/HighestGoodNetworkApp
# Carlos ivy weekly summary hours bar

# Description
- Compare: This change adds functionality to compare data from the last full week with the previous full week.

## Main changes explained:

- check the file totalOrgSummary.jsx.
- Date Functions: New date functions have been implemented to automatically determine the last full week and the week before that.
- Redux Integration: useDispatch from react-redux is now used to fetch complete data for taskProjectHours.
- Visualization: The percent change for the week-over-week comparison is now visualized on the screen

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as admin user
5. go to http://localhost:3000/totalorgsummary
6. verify function Improvement:

![Screenshot 2024-08-06 123906](https://github.com/user-attachments/assets/7c203645-e3ca-4037-a4e5-77fdff5d4465)


7. Dark mode functionality is not included in this PR.


## Note:

While the dates currently aren't dynamically updated, it's recommended to manually change the system date on your machine and restart the application to see data comparisons for different weeks. Dark mode functionality is not included in this PR.
